schema.sql

```diff
@@ -10,4 +10,33 @@
 
```

Comment:
![image](https://github.com/IsmailChangezi/vet-clinic/assets/93039359/803167e8-7e57-4a36-a637-62d3eebca276)

- [ ] It would be nice if you add the following insertion queries into your data.sql file :+1: to follow the project requirements :blush: